assets/app/views/_overview-deployment.html

```diff
@@ -138,7 +139,48 @@
         </div>
 
         <!-- spacer -->
-        <div flex class="visible-xs-block"></div>
+        <div flex></div>
+      </div>
+      <div ng-if="hpa.length" class="hpa-details">
+        Autoscaled:
+        <span class="nowrap">min: {{hpa[0].spec.minReplicas || 1}},</span>
+        <span class="nowrap">
+          max: {{hpa[0].spec.maxReplicas}}
+          <span
```

Comment:
should this set of warnings get moved to a service, at a glance they all get rendered the same way here, could just ng-repeat over an array then.  would also save you from having to set metricsWarning in all of the controllers